README.md

```diff
@@ -1,17 +1,28 @@
-# standards
+# DCMLab/standards
 Repository containing standards developed at the DCML
 
 This repository contains the standard for harmonic annotations developed at the [Digital and Cognitive Musicology Lab](https://dcml.epfl.ch/) at [École Polytechnique Fédérale de Lausanne](https://www.epfl.ch/).
 
-## Use cases
+## Harmonic annotation standard
 
-The annotation standard was used in
+The DCML standard for harmonic annotations consists in a set of [annotation guidelines](https://github.com/DCMLab/standards/blob/master/guidelines/english.md) and a [regular expression](https://github.com/DCMLab/standards/blob/master/harmony.py). The guidelines are the reference for annotators applying the standard for the task of encoding harmonic analysis, and explain how the open-source software [MuseScore](https://musescore.org/) can be used to enter chord labels into scores so that they can be
+* automatically extracted using the tools developed at the DCML, and
+* split into the various encoded features using the regular expression.
 
-1. Moss FC, Neuwirth M, Harasim D, Rohrmeier M (2019) Statistical characteristics of tonal harmony: A corpus study of Beethoven’s string quartets. PLoS ONE 14(6): e0217242. [https://doi.org/10.1371/journal.pone.0217242](https://doi.org/10.1371/journal.pone.0217242)
-2. Neuwirth M, Harasim D, Moss FC and Rohrmeier M (2018) The Annotated Beethoven Corpus (ABC): A Dataset of Harmonic Analyses of All Beethoven String Quartets. Front. Digit. Humanit. 5:16. doi: [https://doi.org/10.3389/fdigh.2018.00016](https://doi.org/10.3389/fdigh.2018.00016)
+### Use Cases
+
+Version 2.0.0 has been used for all annotations commissioned by the DCML since the second half of 2018.
+
+### Log
 
-## Log
+#### v2.0.0
 
-### v1.0.0
-* Initial version of the regular expression to annotate harmonies
-* Taken from: https://github.com/DCMLab/ABC
+* The new version does not allow `9` as `figbass` feature anymore. This means that a label such as `V9` will throw an error. Instead, this chord needs to be written either as `V7(9)` or `V7(+9)`, depending on the context.
```

Comment:
Diese Details sollten in den Guidelines stehen (bzw. nur, wie es *aktuell* gehandhabt wird). Im Log könnte soetwas stehen wie "Removed `9` from `figbass` feature".